code/modules/modular_computers/file_system/programs/generic/file_browser.dm

```diff
@@ -66,7 +67,8 @@
 						"name" = F.filename,
 						"type" = F.filetype,
 						"size" = F.size,
-						"undeletable" = F.undeletable
+						"undeletable" = F.undeletable,
+						"encrypted" = !!F.password
```

Comment:
Not a fan of this kind of obfuscation.

Reply:
This isn't obfuscation, it's boolean conversion. There's not really a more clean way to get the true/false representation of a variable in BYOND.

Reply:
And if you ask the majority of SS13 coders what this means do you think they'll know?

Reply:
But you're right there isn't really a way in BYOND unfortunately.

Reply:
Probably time for us to learn. *shrug*